### PR TITLE
[ja] Update kops versioned download URLs

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/27570/files
/assign @atoato88 
/assign @inductor

### CODE REVIEW

File: content/ja/docs/setup/production-environment/tools/kops.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ To download a specific version, replace the following portion of the command wit
 $(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)
 ```
 
-For example, to download kops version v1.15.0 type:
+For example, to download kops version v1.20.0 type:
 
 ```shell
-curl -LO  https://github.com/kubernetes/kops/releases/download/1.15.0/kops-darwin-amd64
+curl -LO https://github.com/kubernetes/kops/releases/download/1.20.0/kops-darwin-amd64
```
Comment (atoato88) [inline content/ja/docs/setup/production-environment/tools/kops.md:62]: [In original PR](https://github.com/kubernetes/website/pull/27570/files#diff-93e2efd4dc6f52998817f78e73f359bc5c3eaa2aac381b99f281b88a7d5de2edR62), `v` prefix is included to the version tag.
We should updates here with `v1.20.0` similarly?

Comment (atoato88) [inline content/ja/docs/setup/production-environment/tools/kops.md:62]:
```suggestion
curl -LO https://github.com/kubernetes/kops/releases/download/v1.20.0/kops-darwin-amd64
```
Here is my suggestion.

File: content/ja/docs/setup/production-environment/tools/kops.md [hunk 2]
```diff
@@ -94,10 +94,10 @@ To download a specific version of kops, replace the following portion of the com
 $(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)
 ```
 
-For example, to download kops version v1.15.0 type:
+For example, to download kops version v1.20.0 type:
 
 ```shell
-curl -LO  https://github.com/kubernetes/kops/releases/download/1.15.0/kops-linux-amd64
+curl -LO https://github.com/kubernetes/kops/releases/download/1.20.0/kops-linux-amd64
```
Comment (atoato88) [inline content/ja/docs/setup/production-environment/tools/kops.md:100]: The same comment as above.
```suggestion
curl -LO https://github.com/kubernetes/kops/releases/download/v1.20.0/kops-linux-amd64
```

**Comment by atoato88** [comment]
Thank you to creating this! :smile: 
I left some comments, could you check that?

File: content/ja/docs/setup/production-environment/tools/kops.md (hunk 1 above)
Comment (Arhell) [reply content/ja/docs/setup/production-environment/tools/kops.md]: done

File: content/ja/docs/setup/production-environment/tools/kops.md (hunk 2 above)
Comment (Arhell) [reply content/ja/docs/setup/production-environment/tools/kops.md]: done

File: content/ja/docs/setup/production-environment/tools/kops.md (hunk 1 above)
Comment (Arhell) [reply content/ja/docs/setup/production-environment/tools/kops.md]: done

File: content/ja/docs/setup/production-environment/tools/kops.md (hunk 2 above)
Comment (Arhell) [reply content/ja/docs/setup/production-environment/tools/kops.md]: done

**Comment by Arhell** [comment]
@atoato88 all done, еhanks, it looks like when I did the data were old.

**Comment by atoato88** [comment]
/lgtm
Thank you for quick fix.
Perfect! :smile:

**Pull request merged**